exercises/concept/black-jack/.meta/exemplar.py

```diff
@@ -28,7 +28,7 @@ def higher_card(card_one, card_two):
     """Determine which card has a higher value in the hand.
 
     :param card_one, card_two: str - cards dealt. 'J', 'Q', 'K' = 10; 'A' = 1; numerical value otherwise.
-    :return: higher value card - str. Tuple of both cards if they are of equal value.
+    :return: str or tuple - resulting Tuple contains both cards if they are of equal value.
```

Comment:
```suggestion
    :return: str (single card) or tuple (both cards if they are of equal value).
```

Reply:
Don't know if that is helpful or not.  Parenthetical info might be more confusing than being more verbose.  Thoughts?

Reply:
This is one of those "confusing from the outset" return lines where I think it can be helpful to have a readable type and short and sweet description. I'd love for it to look something like:

```
"""
...
:return: str or tuple(str, str) - can be the high card or a tuple of equal valued cards.
"""
or with 'typing' style

"""
...
:return: Union[str, Tuple[str, str]] - can be the high card or a tuple of equal valued cards.
"""
```

My thought process is that the less we re-iterate types, and keep it to just the type section, the less we have to spell things out parenthetically. I like your idea of a short and sweet summary first line and the real meat and potatoes being in the params/return lines.

Also, many of the changes made so far include the dash separating the \<type\> and \<description\> portions of the return/param lines. Are we keeping this as a sort of standard or do we not care as much? This goes back to Isaac's concern about inconsistency in the docstrings.